doc-content/enterprise-only/installation/installer-run-proc.adoc

```diff
@@ -132,12 +132,13 @@ choose to configure database settings or customize certain {KIE_SERVER} options.
 +
 A data source is an object that enables a Java Database Connectivity (JDBC) client, such as an application server, to establish a connection with a database. Applications look up the data source on the Java Naming and Directory Interface (JNDI) tree or in the local application context and request a database connection to retrieve data. You must configure data sources for {KIE_SERVER} to ensure proper data exchange between the servers and the designated database.
 +
-
-. If you selected *Customize {KIE_SERVER} settings*, change any of the following, if desired:
+. If you selected *Customize {KIE_SERVER} properties*, on the *{KIE_SERVER} Properties Configuration*, change any of the following:
 +
-* Change the name of the {KIE_SERVER} property.
-* Change the URL of the {HEADLESS_CONTROLLER}.
-* Deselect any {KIE_SERVER} server options.
+* *{KIE_SERVER} ID*: The name of the {KIE_SERVER} property.
+ifeval::["{context}" == "install-on-eap"]
+* *Controller URL*: The URL of the {HEADLESS_CONTROLLER}.
+endif::[]
+* Clear any {KIE_SERVER} server function you want to disable.
```

Comment:
I would suggest removing **server**, I believe the **server** word must be included with the attribute, please confirm, if it is not added with the attribute then it's fine. Also, suggest adding **that** like ...."Clear any {KIE_SERVER} function that you want to disable."